api/controllers/UserController.js

```diff
@@ -18,12 +19,15 @@ module.exports = {
    * @param      {object}  res     The resource
    */
   createUser:   function createUser(req, res) {
+    var log = Logger.child({'function name': 'createUser'});
```

Comment:
You should define a global log on top of the file

```
var logger = Logger.child({controller: 'UserController'});
```
and inside every action you can do
```
var log = logger.child({action: 'createUser'});
```